<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUser</class>
 <widget class="QWidget" name="FormUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>331</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nIKLabel">
      <property name="text">
       <string>NIK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nIKLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaLabel">
      <property name="text">
       <string>Nama</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="roleLabel">
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="roleComboBox">
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>User</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Form User</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableUser">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>531</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
